app/controllers/coronavirus_form/live_in_england_controller.rb

```diff
@@ -4,12 +4,13 @@ class CoronavirusForm::LiveInEnglandController < ApplicationController
   skip_before_action :check_first_question
 
   def submit
-    live_in_england = strip_tags(params[:live_in_england]).presence
-    session[:live_in_england] = live_in_england
+    @form_responses = {
+    live_in_england: strip_tags(params[:live_in_england]).presence,
+  }
```

Comment:
Nitpick: The indentation is off here.

Reply:
I'm surprised Rubocop didn't pick this up :/.  I've raised https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/234 to fix it.